### PR TITLE
+ Medium API implementation, resolves #9

### LINKED ISSUE #9: Medium API implementation
Medium API to get the user stories to the application.

Documentation - <https://github.com/Medium/medium-api-docs>

### PR DESCRIPTION
**Pull request by xenoworkforce**
There is no way to retrieve Medium posts via their API. Method, described in PR, retrieves RSS-feed which contain blog posts and convert them into JSON.

### CODE REVIEW

File: src/controller/user.controller.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Request, Response, NextFunction } from 'express';
-import { insertUser, getUserById, getUserDetails, updateUserbyId, checkUserByUsernameAndPassword } from '../service/user.service';
+import { insertUser, getUserById, getUserDetails, updateUserbyId, checkUserByUsernameAndPassword, getUserMediumPosts } from '../service/user.service';
```
Comment (rusiruavb) [inline src/controller/user.controller.ts:2]: @rustmn brother, these functions should be in a separate file.
for example, this is for getting Medium posts. So,  it should in a separate file called `medium.controller.ts`

File: src/routes.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Express } from "express";
-import { createUser, loginUser, getUserAccount, getPublicUserAccount, updateUser} from './controller/user.controller';
+import { createUser, loginUser, getUserAccount, getPublicUserAccount, updateUser, getMediumPosts } from './controller/user.controller';
```
Comment (rusiruavb) [inline src/routes.ts:2]: get this `getMediumPosts` from that separate file that I have mention above

**Review by rusiruavb — CHANGES_REQUESTED** [review]
Could you please do the changes that I have mention. Others are LGTM

**Comment by rusiruavb** [comment]
@rustmn Create a separate feature branch by the way.
example: `feature/medium`

**Comment by rusiruavb** [comment]
And also, make the PR to development branch

**Comment by xenoworkforce** [comment]
@rusiruavb, sorry, but I haven't enough time right now, I'll look here as I'm free.

**Comment by rusiruavb** [comment]
> @rusiruavb, sorry, but I haven't enough time right now, I'll look here as I'm free.

Ah.. it's fine @rustmn . Actually, thank you for solving my problem. 😍❤

**Pull request merged**